website/src/pages/InnerSource.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Is this a special layout that cannot be achieved through `md / mdx`?

Reply:
yes - with MDX or MD you get a left-hand navigator, which was something I wanted to avoid on the landing pages.